.github/workflows/backend_tests.yml

```diff
@@ -0,0 +1,38 @@
+name: Backend tests
+
+on:
+  push:
+  pull_request:
+    types: [opened, reopened]
+
+jobs:
+  tests:
+
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        os: [ubuntu-18.04, ubuntu-latest]
+        python-version: [3.6, 3.7, 3.8]
+        mongodb-version: [4.2]
```

Comment:
That is not what we're using though: 
https://github.com/research-software-directory/research-software-directory/blob/f5861de2c4b5373300110fc0dc08febb505a9709/database/Dockerfile#L1

Reply:
Using only Mongodb 3.6